src/utils/projects.ts

```diff
@@ -10,15 +12,14 @@ export async function fetchFeaturedProjects<T>() {
   explorerUrlFeatured.searchParams.set('isFeatured', 'true')
 
   try {
-    const discoverProjectsData = await fetch(explorerUrlFeatured)
-      .then(async res => res.json())
-      .catch(err => console.error({ featuredProjects: err }))
+    const discoverProjectsData = await fetch(explorerUrlFeatured).then(async res => res.json())
     const discoverProjects = Object.values(discoverProjectsData.projects)
 
     return {
       data: discoverProjects as T
     }
   } catch (error) {
+    logError(error)
```

Comment:
Not necessary here bc this is already a rethrow in the line below and we're seeing this in Sentry. Might cause dupes.


```suggestion
```